keps/sig-node/1977-container-notifier/README.md

```diff
@@ -312,11 +311,11 @@ type PodNotificationErrorType string
 
 // More error types could be added, e.g., Forbidden, Unauthorized, etc.
 const (
-    // Pod does not exist
-    NonExistingPod PodNotificationErrorType = "NonExistingPod"
+    // Pod not found error
+    PodNotFound PodNotificationErrorType = "PodNotificationErrorPodNotFound"
```

Comment:
You changed this the opposite of what I meant :)

`PodNotifictionErrorPodNotFound PodNotificationErrorType = "PodNotFound"`

Reply:
Ack